.github/workflows/update-apidog-dev.yml

```
@@ -0,0 +1,35 @@
+      - name: Checkout code
+        uses: actions/checkout@v4  # 최신 버전 사용
+      - name: Set up JDK 21
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: '21'
+      - name: Build project with Gradle
+        run: ./gradlew build
+      - name: Run Spring Boot app in background
+        run: |
+          nohup java -jar build/libs/echo-bloom-server-0.0.1-SNAPSHOT.jar &  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)